Tests/AnyLintTests/StatisticsTests.swift

```diff
@@ -7,91 +7,105 @@
-        Statistics.shared.found(
-            violations: [Violation(checkInfo: checkInfo1)],
-            in: checkInfo1
-        )
+        Statistics.default.found(violations: [checkInfo1: [Violation(checkInfo: checkInfo1)]])
 
-        XCTAssertEqual(Statistics.shared.executedChecks, [checkInfo1])
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.info]!.count, 1)
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.warning]!.count, 0)
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.error]!.count, 0)
-        XCTAssertEqual(Statistics.shared.violationsPerCheck.keys.count, 1)
+        XCTAssertEqual(Statistics.default.executedChecks, [checkInfo1])
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.info]!.count, 1)
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.warning]!.count, 0)
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.error]!.count, 0)
+        XCTAssertEqual(Statistics.default.violationsPerCheck.keys.count, 1)
 
         let checkInfo2 = CheckInfo(id: "id2", hint: "hint2", severity: .warning)
-        Statistics.shared.found(
-            violations: [Violation(checkInfo: checkInfo2), Violation(checkInfo: checkInfo2)],
-            in: CheckInfo(id: "id2", hint: "hint2", severity: .warning)
+        Statistics.default.found(
+            violations: [
+                CheckInfo(id: "id2", hint: "hint2", severity: .warning):
+                    [Violation(checkInfo: checkInfo2), Violation(checkInfo: checkInfo2)]
+            ]
         )
 
-        XCTAssertEqual(Statistics.shared.executedChecks, [checkInfo1, checkInfo2])
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.info]!.count, 1)
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.warning]!.count, 2)
-        XCTAssertEqual(Statistics.shared.violationsBySeverity[.error]!.count, 0)
-        XCTAssertEqual(Statistics.shared.violationsPerCheck.keys.count, 2)
+        XCTAssertEqual(Statistics.default.executedChecks, [checkInfo1, checkInfo2])
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.info]!.count, 1)
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.warning]!.count, 2)
+        XCTAssertEqual(Statistics.default.violationsBySeverity[.error]!.count, 0)
+        XCTAssertEqual(Statistics.default.violationsPerCheck.keys.count, 2)
 
         let checkInfo3 = CheckInfo(id: "id3", hint: "hint3", severity: .error)
-        Statistics.shared.found(
-            violations: [Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3)],
-            in: CheckInfo(id: "id3", hint: "hint3", severity: .error)
+        Statistics.default.found(
+            violations: [
+                CheckInfo(id: "id3", hint: "hint3", severity: .error):
+                [Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3), Violation(checkInfo: checkInfo3)]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)